### DISCUSSION TITLE
Docs and examples

### DISCUSSION THREAD

**Discussion by hanifshahy** [DinoLeung/TeleDart#271] [Ideas]
Provide more examples and add more docs in README.md file.

**Comment by DinoLeung** [comment]
there's an example [here](https://github.com/DinoLeung/TeleDart/blob/master/example/main.dart) currently, although it's rather minimal.

**Comment by paulobreim** [comment]
Dino,
Why import 'dart:io'; can´t be used with modal and need changa like import 'dart:io as io' for example?

**Reply by DinoLeung** [reply]
it can be just `import 'dart:io'` in the code, but it's just my preference to give an alias so the class name in my telegram models don't conflict with models in io library, e.g. file
